x-pack/plugins/observability/public/components/app/observability_status/observability_status_box.test.tsx

```diff
@@ -35,25 +38,28 @@ describe('ObservabilityStatusBox', () => {
     });
 
     it('should have a learn more button', () => {
-      const learnMoreLink = screen.getByRole('link') as HTMLAnchorElement;
-      expect(learnMoreLink.href).toContain('learnMoreUrl.com');
+      const learnMoreLink = screen.getByText('Learn more') as HTMLAnchorElement;
```

Comment:
is the type casting here still correct?

Reply:
Good catch! It wasn't correct, change it to `HTMLElement`